timelock-impl/build.gradle

```diff
@@ -70,7 +67,9 @@ dependencies {
     testImplementation 'com.palantir.sls.versions:sls-versions'
     testImplementation 'com.palantir.tokens:auth-tokens'
     testImplementation 'com.palantir.tritium:tritium-registry'
+    testImplementation 'com.zaxxer:HikariCP'
     testImplementation 'io.dropwizard.metrics:metrics-core'
+    implementation 'io.undertow:undertow-core'
```

Comment:
ideally we'd push this to join the rest of the normal implementation ones?

Reply:
Done